### PR TITLE
Fix RDEPENDS in pidgin recipe for finch

### PR DESCRIPTION
**Pull request by muhamzai**

### CODE REVIEW

**Comment by kergoth** [comment]
@muhamzai This isn't the right way to change a recipe. You don't need to copy the recipe and patches, just create a bbappend for it. See the yocto project or mel flex product documentation, or look elsewhere in this layer for examples.

**Review by kergoth — CHANGES_REQUESTED** [review]
See previous comment.

**Comment by muhamzai** [comment]
> @muhamzai This isn't the right way to change a recipe. You don't need to copy the recipe and patches, just create a bbappend for it. See the yocto project or mel flex product documentation, or look elsewhere in this layer for examples.

All right, though if we need to send an upstream patch to add a runtime dependency should we still create a bbappend for us?

**Comment by muhamzai** [comment]
@kergoth Updated the PR

**Comment by Noor-Ahsan** [comment]
@kergoth kindly reviw the changes so that we can merge this

**Review by ArsalanHAwan — APPROVED** [review]
LGTM.

Acked-by: Arsalan H. Awan <Arsalan_Awan@mentor.com>

File: meta-sokol-flex-staging/dynamic-layers/openembedded-layer/recipes-support/pidgin/pidgin_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: GPL-2.0
+# ---------------------------------------------------------------------------------------------------------------------
+
+RDEPENDS:${PN}:append = " python3"
```
Comment (ArsalanHAwan) [inline meta-sokol-flex-staging/dynamic-layers/openembedded-layer/recipes-support/pidgin/pidgin_%.bbappend:5]: @muhamzai I have worked with @akif-tariq to debug and determine that we need to set the runtime dependency for the finch package, not the ${PN}.
Can you please update this as `RDEPENDS:finch:append = " python3"`

**Review by ArsalanHAwan — CHANGES_REQUESTED** [review]

File: meta-sokol-flex-staging/dynamic-layers/openembedded-layer/recipes-support/pidgin/pidgin_%.bbappend (hunk 1 above)
Comment (ArsalanHAwan) [reply meta-sokol-flex-staging/dynamic-layers/openembedded-layer/recipes-support/pidgin/pidgin_%.bbappend:5]: @kergoth or should it be `RDEPENDS:finch:append = " python3-core"` ?

Comment (ArsalanHAwan) [reply meta-sokol-flex-staging/dynamic-layers/openembedded-layer/recipes-support/pidgin/pidgin_%.bbappend:5]: @muhamzai I think it should be `RDEPENDS:finch:append = " python3-core"` like we saw in the bluez recipe.

Comment (muhamzai) [reply meta-sokol-flex-staging/dynamic-layers/openembedded-layer/recipes-support/pidgin/pidgin_%.bbappend]: @ArsalanHAwan updated the PR as the convention we saw in bluez5 
@kergoth kindly review

**Review by ArsalanHAwan — APPROVED** [review]
LGTM.

Acked-by: Arsalan H. Awan <Arsalan_Awan@mentor.com>

**Review by kergoth — APPROVED** [review]
I don't know anything about finch or the underlying issue, but the change itself looks reasonable.

**Pull request merged**